deps/build.jl

```diff
@@ -0,0 +1,28 @@
+using BinDeps
+using Compat
+
+@BinDeps.setup
+
+libmysqlclient = library_dependency("libmysqlclient",
+                                    alias=["libmysqlclient.so",
```

Comment:
`aliases`, and don't need to list the so or dylib (bindeps knows how to find the extensions already)